app/src/main/res/layout/fragment_random.xml

```diff
@@ -36,7 +36,7 @@
 		android:layout_height="wrap_content"
 		android:layout_margin="@dimen/spacing_medium_more"
 		android:gravity="bottom|center_horizontal"
-		android:text="@string/example_wolox"
+		android:text="Example Wolox"
```

Comment:
I guess this change can be undone since you dont need to make changes in this file